docs/contributor/resources/01-kyma.md

```diff
@@ -81,6 +81,18 @@ spec:
 > **CAUTION:**
 > Module referencing using NamespacedName and FQDN (Fully Qualified Domain Name) has been deprecated.
 
+### **.spec.modules[].managed**
+
+The **managed** field determines whether or not the Lifecycle Manager manages the module. By default, it is set to `true`. By setting it to `false`, the module is opted out of management by Lifecycle Manager. This means that the module and all its related resources remain in the remote cluster in the same state they were in when the module became unmanaged. From this time on, Lifecycle Manager does NOT reconcile the module and its resources.
```

Comment:
```suggestion
The **managed** field determines whether or not the Lifecycle Manager manages the module. By default, it is set to `true`. By setting it to `false`, the module is opted out of management by Lifecycle Manager. This means that the module and all its related resources remain in the remote cluster in the same state they were in when the module became unmanaged. From this time on, Lifecycle Manager does NOT reconcile the module and its resources. In addition, the labels `operator.kyma-project.io/managed-by=kyma` and `operator.kyma-project.io/watched-by=kyma` are removed from the module's resources. This may be relevant if you for instance use those labels as filters for setting up telemetry auto-instrumentation.
```

Reply:
As quickly discussed with Icke.